### PR TITLE
gh-1333 separate msgpack extensions from box_protocol

### PR DESCRIPTION
**Pull request by Onvember**

### CODE REVIEW

File: doc/reference/reference_lua/box_info.rst [hunk 1]
```diff
@@ -280,7 +280,7 @@ variables.
       vector clock table.
       Also the replica sends its entire vector clock table in response
       to a master's heartbeat message, see the heartbeat-message examples
-      in section :ref:`Binary protocol -- replication <box_protocol-replication>`.
+      in section :ref:`Binary protocol -- replication <box_protocol-heartbeat>`.
```
Comment (lenkis) [inline doc/reference/reference_lua/box_info.rst:283]: are you sure about the link label?

Comment (Onvember) [reply doc/reference/reference_lua/box_info.rst:283]: yes, I've added this tag [here](https://github.com/tarantool/doc/pull/1366/files#diff-90b504e032fff8895ef9e66d545c19f2R1013)

File: locale/ru/LC_MESSAGES/dev_guide/internals/msgpack_extensions.po [hunk 2]
```diff
@@ -10,37 +10,52 @@ msgid ""
 "default MessagePack types, and correctly mapped to programming language "
 "types."
 msgstr ""
+"Tarantool использует предопределенные типы расширений MessagePack для "
+"представления некоторых специальных значений. Типы расширений включают "
+"``MP_DECIMAL``, ``MP_UUID`` и ``MP_ERROR``. Эти типы требуют особого "
+"внимания со стороны разработчиков коннекторов, так как должны "
+"рассматриваться отдельно от типов MessagePack по умолчанию, и корректно "
```
Comment (lenkis) [inline locale/ru/LC_MESSAGES/dev_guide/internals/msgpack_extensions.po:17]:
```suggestion
"рассматриваться отдельно от типов MessagePack по умолчанию и корректно "
```

File: locale/ru/LC_MESSAGES/dev_guide/internals/msgpack_extensions.po [hunk 3]
```diff
@@ -124,29 +160,37 @@ msgid ""
...
 
 msgid ""
 "The `MessagePack spec "
 "<https://github.com/msgpack/msgpack/blob/master/spec.md>`_ defines ``d8``"
 " to mean fixext with size 16, and a uuid's size is always 16. So the uuid"
 " MessagePack representation looks like this:"
 msgstr ""
+"`Спецификация MessagePack "
+"<https://github.com/msgpack/msgpack/blob/master/spec.md>`_ определяет "
+"``d8`` как ``fixext`` с размером 16, а размер ``uuid всегда 16. Таким "
```
Comment (lenkis) [inline locale/ru/LC_MESSAGES/dev_guide/internals/msgpack_extensions.po:192]:
```suggestion
"``d8`` как ``fixext`` с размером 16, а размер ``uuid`` всегда равен 16. Таким "
```

File: locale/ru/LC_MESSAGES/dev_guide/internals/msgpack_extensions.po [hunk 4]
```diff
@@ -177,23 +228,33 @@ msgid ""
...
+"following what was described in :ref:`Box protocol -- responses for "
+"errors <box_protocol-responses_error>`. This is a \"compatible\" "
+"enhancement, because clients that expect old-style server responses "
+"should ignore map components that they do not recognize. Notice, however,"
+" that there has been a renaming of a constant: formerly IPROTO_ERROR in "
+"./box/iproto_constants.h was 0x31, now IPROTO_ERROR is 0x52 and "
+"IPROTO_ERROR_24 is 0x31."
+msgstr ""
+"Начиная с версии 2.4.1, в ответах на ошибки содержится дополнительная "
+"информация, следующая описанию в разделе :ref:`Box protocol -- responses "
```
Comment (lenkis) [inline locale/ru/LC_MESSAGES/dev_guide/internals/msgpack_extensions.po:251]:
```suggestion
"информация, соответствующая описанию в разделе :ref:`Box protocol -- responses "
```

File: locale/ru/LC_MESSAGES/dev_guide/internals/msgpack_extensions.po [hunk 5]
```diff
@@ -209,61 +270,94 @@ msgid ""
...
+"mp_error.cc."
 
 msgid ""
 "For example, in version 2.4.1 or later, if we try to create a duplicate "
 "space with |br| ``conn:eval([[box.schema.space.create('_space');]])`` "
 "|br| the server response will look like this:"
 msgstr ""
+"Например, в версии 2.4.1 или более поздней, если мы попытаемся создать "
+"дубликат пробела с помощью команды |br| "
+"``conn:eval([[box.schema.space.create('_space');]))`` |br| ответ сервера "
```
Comment (lenkis) [inline locale/ru/LC_MESSAGES/dev_guide/internals/msgpack_extensions.po:359]:
```suggestion
"``conn:eval([[box.schema.space.create('_space');]))``, |br| ответ сервера "
```

**Pull request merged**